sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -0,0 +1,651 @@
+
+IMPORTANT: In the current implementation, only the static extent is supported
+
+#### Use
+Specifying the usage of the matrix: matrix left (A), matrix right (B) or accumuator (C) is required by backend implementations to reason about the layout of the matrix in regiters.
+
+```c++
+namespace sycl::ext::oneapi::experimental::matrix {
+enum class matrix_use {
+  a,
+  b,
+  accumulator,
+};
+}
+```
+
+IMPORTANT: In both AMX and DPAS support, `use` template parameter is required
+
+
+#### Layout
+Besides row major and column major layouts, `matrix_layout` is flexible enough to introduce customed layouts such as symmetric or tiled layouts.
+	
+```c++
+namespace sycl::ext::oneapi::experimental::matrix {
+enum class matrix_layout {
+  row_major,
+  col_major,
+  packed,
+  unused,
+};
+}
+```
+
+IMPORTANT: In both AMX and DPAS support, layout template parameter is unused and will be ignored if specified
+
+#### Memory Scope
+In this experimental API version, we used the terminology of `joint_matrix` instead of plain `matrix` to emphasis that the matrix is shared among a group of work items and is not private to each work item. The memory scope is added as an additional template parameter and is also part of the constructor arguments.
+
+IMPORTANT: In the current implementation, only the subgroup scope is supported
+
+When the group is a `sycl::sub_group`, a matrix is declared as follows:
+
+```c++
+joint_matrix<int8_t, tM, tN, matrix_use::matrix_a> tA(sg); 
+```
+
+
+## Matrix Operations and their Execution Scope
+We define three new functions needed to perform the main and common operations on matrices namely, load, store, and the actual multiply and add operation. This set of functions can be easily extended if the tensor hardware implements new features.
+
+The base pointer determines the starting address of the matrix to be loaded/stored. `layout` determines whether the data are being read/written in a row (`row_major`), column major (`column_major`) fashion, or if the data has already been transformed into VNNI format (`packed`). `stride` describes the number of elements between consecutive rows for row major and packed layout,  columns for column major layout. 
+
+Note that for getting maximum performance on Intel AMX and DPAS, prepacking data in the memory is necessary. If users did not specify the packed layouts, transforms done by the implementation will be slow due to extra scatter/gather operations. Hence, we expose the `packed` layout to the user to specify that A or B have already been VNNIed. The packed or VNNI layout is introduced in `VNNI layout` section below.
+	
+IMPORTANT: In the current AMX and DPAS implementation, the layout in the load of matrix B must be `packed_b` or `row_major`. Automatic VNNI transform is supported. The layout in the load of matrices A and C must be `row_major`, and the layout in the store of matrix C must also be `row_major`.
```

Comment:
```suggestion
IMPORTANT: In the current AMX and DPAS implementation, the layout in the load of matrix B must be `packed` or `row_major`. Automatic VNNI transform is supported. The layout in the load of matrices A and C must be `row_major`, and the layout in the store of matrix C must also be `row_major`.
```

I guess if you are removing `packed_b` and `packed_a` this sentence will change like this?